mobile/src/screens/HomeScreen.tsx

```diff
@@ -52,15 +52,15 @@ export const HomeScreen: React.FC = () => {
       </View>
 
       <View>
-        <HorizontalBookList books={newBooks} />
+        { booksCtx.loading ? <ActivityIndicator/> : <HorizontalBookList books={newBooks}/> }
```

Comment:
The default props for `<ActivityIndicator>` aren't visible on Android :(

I played with it a little on my phone and found that `<ActivityIndicator size="large" color={Colors.orange}/>` looked best on Android, but I don't know if that would look good on iOS too.

Can you guys add a `<LoadingCircle>` component to `src/components`, wrapping an ActivityIndicator for consistency across the app? If size="large" and color=Colors.orange looks good on iOS you can just drop that right into the return statement, but if some other color (or the default color) looks better you can use the [Platform module](https://reactnative.dev/docs/platform-specific-code) to customize it for each OS. Definitely make it `size="large"` on iOS though for consistency.